### ISSUE TITLE
Ongoing runtime lib

### ISSUE THREAD

**Issue by Anatidae-Project** [eclipsesource/J2V8#71]
In the case where I have a large JavaScript library that also contains some statefulness, would I not close that runtime until I was finished?

I'm working in Android, so in my main Activity I'd load my script and I assume run runtime.executeScript(myScript);

In this case, my script is a large 2000+ line collection of libraries with the core functionality maintaining some statefulness. I am assuming if I runtime.release() the script I loaded is discarded? Or is it that once I load it in there, it's ready for running. 

Basically, I want to load this lib and call function and also set up the lib to fire off events I'll set up listeners for.

**Comment by irbull** [comment]
Do not release your runtime until your system is shutting down. Once you release it, you cannot use it again. I suggest either using J2V8 3.0 or looking at the sources on GitHub as I've documented all the API now.

I typically runtime.executeScript(myscript); and then use runtime.callFunction() to access the functionality in JS.
